app/Helpers/render/site-award.php

```diff
@@ -139,8 +139,8 @@ function RenderAwardGroup(array $awards, string $title, string $awardsOwnerUsern
 
     echo "<div id='" . strtolower(str_replace(' ', '', $title)) . "'>";
     echo "<h3 class='flex justify-between gap-2'><span class='grow'>$title</span>$counters</h3>";
-    echo "<div class='flex justify-center'>";
-    echo "<div class='component flex flex-wrap gap-2 justify-start bg-embed w-full xl:rounded xl:py-2 xl:px-5'>";
+    echo "<div class='max-sm:flex max-sm:justify-center'>";
+    echo "<div class='component w-full bg-embed gap-2 grid grid-cols-auto-fill-award-grid max-sm:grid-cols-5 max-sm:max-w-xs max-sm:px-4 max-sm:py-2 lg:gap-x-0 xl:py-2 xl:px-4'>";
```

Comment:
Something is adding roughly 0.6rem of bottom margin to the `<h3>`, only on XS. Let's remove this for now so the spacing between the header and content is visually consistent across all breakpoints.

Reply:
I assume this is being caused by the max-sm:py-2 that I added to match the XL breakpoint. This has now been removed.